src/utils/useLocationUpdateEffect.ts

```diff
@@ -1,188 +0,0 @@
-import firebase, { UserInfo } from "firebase/app";
-
-import { AnyRoom } from "types/rooms";
-import { AnyVenue, VenueEvent } from "types/venues";
-import { User } from "types/User";
-
-import { updateUserProfile } from "pages/Account/helpers";
-import { useInterval } from "hooks/useInterval";
-
-import { WithId } from "./id";
-import { getCurrentTimeInMilliseconds } from "./time";
-import { openRoomUrl, openUrl, venueInsideUrl } from "./url";
-
-const LOCATION_INCREMENT_SECONDS = 10;
-const LOCATION_INCREMENT_MS = LOCATION_INCREMENT_SECONDS * 1000;
-
-export const updateLocationData = (
-  user: UserInfo,
-  locationName: { [key: string]: number },
-  lastSeenIn: { [key: string]: number } | undefined
-) => {
-  const location = locationName ?? {};
-  const prevRoomName =
-    lastSeenIn &&
-    Object.keys(lastSeenIn).find((lastSeen) => lastSeen.includes("/"));
-
-  if (lastSeenIn && prevRoomName) {
-    delete lastSeenIn[prevRoomName];
-  }
-
-  const roomVenue =
-    locationName && Object.keys(locationName).length
-      ? Object.keys(locationName)[0]
-      : null;
-
-  updateUserProfile(user.uid, {
-    lastSeenAt: getCurrentTimeInMilliseconds(),
-    lastSeenIn:
-      !locationName && !lastSeenIn
-        ? null
-        : lastSeenIn
-        ? { ...lastSeenIn, ...location }
-        : location,
-    room: !locationName && !lastSeenIn ? null : roomVenue,
-  });
-};
-
-// get Profile from the firebase
-// @debt rename this trackLocationEntered?
-export const trackRoomEntered = (
-  user: UserInfo,
-  locationName: { [key: string]: number },
-  lastSeenIn: { [key: string]: number } | undefined
-) => {
-  updateLocationData(user, locationName, lastSeenIn);
-};
-
-export interface TrackRoomEnteredNGProps {
-  user?: UserInfo;
-  venue: AnyVenue;
-  room: AnyRoom;
-  lastSeenIn?: Record<string, number>;
-}
-
-export const trackRoomEnteredNG = ({
-  user,
-  venue,
-  room,
-  lastSeenIn,
-}: TrackRoomEnteredNGProps) => {
-  if (!user) return;
-
-  trackRoomEntered(
-    user,
-    { [`${venue.name}/${room.title}`]: getCurrentTimeInMilliseconds() },
-    lastSeenIn
-  );
-};
-
-export interface TrackVenueEnteredProps {
-  user?: UserInfo;
-  venue: AnyVenue;
-  lastSeenIn?: Record<string, number>;
-}
-
-export const trackVenueEntered = ({
-  user,
-  venue,
-  lastSeenIn,
-}: TrackVenueEnteredProps) => {
-  if (!user) return;
-
-  trackRoomEntered(
-    user,
-    { [venue.name]: getCurrentTimeInMilliseconds() },
-    lastSeenIn
-  );
-};
```

Comment:
Why was this removed?

Reply:
https://github.com/sparkletown/sparkle/pull/1108#discussion_r567819017

Reply:
https://github.com/sparkletown/sparkle/pull/1108#discussion_r568322817